R/broom-package.R

```diff
@@ -1,13 +1,17 @@
-#' @title Convert Statistical Objects into Tidy Tibbles
-#' @name broom
-#' @description Convert statistical analysis objects from R into tidy tibbles,
-#'   so that they can more easily be combined, reshaped and otherwise processed
-#'   with tools like dplyr, tidyr and ggplot2. The package provides three S3
-#'   generics: tidy, which summarizes a model's statistical findings such as
-#'   coefficients of a regression; augment, which adds columns to the original
-#'   data such as predictions, residuals and cluster assignments; and glance,
-#'   which provides a one-row summary of model-level statistics.
-#'
+# Custom package description for the package help file.
+#' @description
+#' Convert statistical analysis objects from R into tidy tibbles,
+#' so that they can more easily be combined, reshaped and otherwise processed
+#' with tools like dplyr, tidyr and ggplot2. The package provides three S3
+#' generics: tidy, which summarizes a model's statistical findings such as
+#' coefficients of a regression; augment, which adds columns to the original
+#' data such as predictions, residuals and cluster assignments; and glance,
+#' which provides a one-row summary of model-level statistics.
+#' 
```

Comment:
```suggestion
#' 
#' @aliases broom broom-package
#' 
```

So that folks can still `?broom` and that dead link will still work.

Reply:
As you can see in the .Rd file, it still keeps the broom alias. So ?broom will work.

For keeping the broom link, I can add `@name broom`